test/tests/functional/pbs_offline_vnodes.py

```diff
@@ -90,26 +83,28 @@ def test_single_mom_hook_failure_affects_vnode(self):
         Once offlined, reset the mom by issueing pbsnodes -r
         and check if the job runs on one of the vnodes.
         """
-        self.create_multi_vnodes(1)
+        single_mom = ""
+        if len(self.moms.values()) != 1:
+            self.server.manager(MGR_CMD_DELETE, NODE, id="@default",
+                                expect=True)
+        single_mom = self.moms.values()[0]
+
+        cpuset_mom = single_mom.is_cpuset_mom()
+        if cpuset_mom is False:
+            self.create_multi_vnodes(1)
         self.create_mom_hook()
-        # Remove the second node from the system (if any)
-        try:
-            self.server.manager(MGR_CMD_DELETE, NODE,
-                                id=self.moms.values()[1].shortname)
-        except PbsManagerError as e:
-            self.assertTrue("Unknown node" in e.msg[0])
 
         j1 = Job(TEST_USER)
         self.server.expect(NODE, {ATTR_NODE_state: 'free'},
-                           id=self.moms.values()[0].shortname, max_attempts=3,
+                           id=single_mom.shortname, max_attempts=3,
                            interval=2)
         j1.set_sleep_time(1000)
         jid = self.server.submit(j1)
         self.server.expect(JOB, {ATTR_state: 'Q'}, id=jid)
 
         self.server.expect(
             NODE, {ATTR_NODE_state: 'offline'},
-            id=self.moms.values()[0].shortname, max_attempts=3, interval=2)
+            id=single_mom.shortname, max_attempts=20, interval=2)

```

Comment:
Earlier you had expected the check to be quick and that's why max_attempts=3.  But now it is set to 20.  Is that because you added UV testing?

Reply:
Yes Lisa this is because of UV testing. I found out that it took longer on a UV to set the nodes to offline state. Not sure why it was so but thought it is better to increase the number of attempts.

Reply:
Okay.  Thanks for confirming.